src/components/AmountInput.tsx

```diff
@@ -0,0 +1,36 @@
+import clsx from 'clsx';
+
+type InputProps = {
```

Comment:
not sure why this is called `InputProps` instead of merely `Props`. If it were `AmountInputProps` I'd understand it as being specific about the component. Though seems redundant in an `AmountInput` module with one other binding.

Reply:
Sure changed to just `Props`